addon/bin/sign

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+const version = require('../package.json').version;
+const resolve = require('path').resolve;
+const xpiPath = resolve(`dist/testpilot-addon-${version}.xpi`);
```

Comment:
> Ahh I didn't realize we were on that old of a node version. I'll pull the template strings out.

Yeah, the version [is pinned in the Dockerfile](https://github.com/mozilla/testpilot/blob/master/Dockerfile-frontend-build#L1) for the frontend. We should probably upgrade that, but not as a part of this PR.

I also [filed an issue](https://github.com/mozilla-jetpack/jpm/issues/505) on `jpm sign` and trying to use it repeatedly on the same version. When I get around to trying CircleCI integration, it sounds like I might need to drop to the HTTP API instead of using the CLI tool :disappointed: